app/environments/brassbirmingham/brassbirmingham/envs/classes/buildings/building.py

```diff
@@ -80,7 +80,7 @@ def build(self, buildLocation):
     def sell(self):
         self.isActive = False
         self.isSold = True
-        self.isRetired = True
+        # self.isRetired = True
```

Comment:
Should we just remove this line if it isn't needed?

Reply:
Yes